lms/static/js/staff_debug_actions.js

```diff
@@ -127,4 +136,8 @@ $(document).ready(function() {
         StaffDebug.rescore($(this).parent().data('location-name'), $(this).parent().data('location'));
         return false;
     });
+    $courseContent.on("click", '.staff-debug', function() {
+        StaffDebug.check_sudo_and_view_modal($(this).data('element-id'), $(this).data('is-sudo'));
+        return false;
+    });
```

Comment:
I don't think that the staff-debug view should be protected by sudo-mode, because the staff-debug information is displayed inside the courseware (and even staff users shouldn't have sudo-mode activated when viewing courseware).